tetrateci/create_istio_release.sh

```diff
@@ -44,6 +44,9 @@ if [[ ${BUILD} == "fips" ]]; then
 fi
 
 # HACK : default manifest from release builder is modified
+export BUILD_WITH_CONTAINER=0
```

Comment:
Move this to just below ISTIO_VERSION. Add a comment saying 
## exporting this environment so that release-builder and Istio build would run outside build container to use custom golang set up